lib/src/key/bc.rs

```diff
@@ -0,0 +1,59 @@
+use crate::idx::ft::terms::TermId;
+use derive::Key;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Key)]
+pub struct Bc<'a> {
+	__: u8,
+	_a: u8,
+	pub ns: &'a str,
+	_b: u8,
+	pub db: &'a str,
+	_c: u8,
+	pub tb: &'a str,
+	_d: u8,
+	_e: u8,
+	_f: u8,
+	pub ix: &'a str,
+	_g: u8,
+	pub term_id: TermId,
+}
+
+impl<'a> Bc<'a> {
+	pub fn new(ns: &'a str, db: &'a str, tb: &'a str, ix: &'a str, term_id: TermId) -> Self {
+		Self {
+			__: 0x2f, // /
```

Comment:
You should use `'/'` characters as they are easier to reason about. There is also a shared definition of these characters somewhere, but can't remember where (key/idx.rs maybe for reference)

Reply:
Im guessing it may make sense to merge with main, as that change is part of main branch.

Reply:
Relevant point! I saw that you switched to bytes literal @phughk, this is very elegant!
I changed to bytes literals everywhere.
I also fixed some consistency issues in the new keys.